File: algorithm/src/main/java/baekjoon/p10864/Main.java
package baekjoon.p10864;

import java.util.Scanner;

public class Main {

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int n = sc.nextInt();
        int m = sc.nextInt();
        int[][] inputs = new int[m][2];

        for (int i = 0; i < inputs.length; i++) {
            for (int j = 0; j < inputs[i].length; j++) {
                inputs[i][j] = sc.nextInt();
            }
        }

        for (int solve : solve(n, inputs)) {
            System.out.println(solve);
        }
    }

    private static int[] solve(int n, int[][] inputs) {
        int[] ret = new int[n];

        for (int[] input : inputs) {
            ret[input[0] - 1]++;
            ret[input[1] - 1]++;
        }

        return ret;
    }

}

File: algorithm/src/main/java/baekjoon/p10864/Main.kt
package baekjoon.p10864

fun main(args: Array<String>) {
    val (n, m) = readLine()!!.split(" ").map { it.toInt() }
    val inputs = List(m) { readLine()!!.split(" ").map { it.toInt() } }
    println(solve(n, inputs).joinToString("\n"))
}

private fun solve(n: Int, inputs: List<List<Int>>): Array<Int> {
    val ret = Array(n) { 0 }
    for (input in inputs) {
        ret[input[0] - 1]++
        ret[input[1] - 1]++
    }

    return ret
}